### DISCUSSION TITLE
Partial Syntax Highlighting of Function Names in R Files on GitHub

### DISCUSSION THREAD

**Discussion by tanchangde** [github-linguist/linguist#6704] [Classification] [answered]
In reviewing R files on GitHub, I've noticed an issue where the initial letters of some function names are not highlighted, contrary to the rest of the function name. 

**Steps to Reproduce:**

1. For an example, see the file: [httr2/R/req-body.R](https://github.com/r-lib/httr2/blob/main/R/req-body.R).
2. Observe the inconsistent syntax highlighting, particularly how the initial letters of some function names are not highlighted as the rest of the name.

**Expected Behavior:**

The entire function name, including its initial letter, should be uniformly highlighted to ensure readability and consistency across R scripts hosted on GitHub.

**Actual Behavior:**

There is a noticeable inconsistency in the highlighting of function names, specifically, the initial letters of some names are not highlighted, while the rest of the name is.

**Additional Information:**

- This issue is seen in the GitHub web interface, which suggests it may be related to how GitHub's syntax highlighting engine processes R language syntax.
- This highlighting inconsistency has been observed in multiple R files, indicating a broader issue with the syntax highlighting rules applied to R scripts on GitHub.

**Comment by lildude** [comment, marked as answer]
This is the same issue reported in https://github.com/github-linguist/linguist/discussions/6505. As with that one, please contact GitHub support as this is not a problem with linguist or the grammar.

**Reply by tanchangde** [reply]
Thank you for your response. Someone has already provided feedback in the GitHub community forums(https://github.com/orgs/community/discussions/109186), but I'm not sure if it's appropriate.
